src/main/scala/com/evojam/play/elastic4s/core/crud/package.scala

```diff
@@ -0,0 +1,20 @@
+package com.evojam.play.elastic4s.core
+
+import play.api.libs.json.{Json, Reads}
+
+import org.elasticsearch.action.get.GetResponse
+
+package object crud {
+
+  implicit class RichGetResponse(response: GetResponse) {
+
+    def parseJson[T: Reads]: Option[T] =
+      Some(response)
+        .filter(_.isExists)
+        .map(_.getSourceAsBytes)
+        .map(Json.parse)
+        .map(_.as[T])
```

Comment:
I'd assume that if someone is aware of inconsistent schema, they should provide a proper Reads instance (e.g. `Reads[Either[OldApiEntity, NewApiEntity]]`). Throwing an exception is a sane behavior.